community/blog/2022-12 Rethinking event publishing in Arcus EventGrid v3.3/README.md

```diff
@@ -0,0 +1,89 @@
+            "<my-authentication-secret-name>");
+    });
+}
+```
+
+Event publishing happens the same way as before. With Azure clients, you can inject such a publisher instance with ease.
+
+```csharp
+using Azure.Messaging.EventGrid;
+using Microsoft.Extensions.Azure;
+
+public class MyService
+{
+    public MyService(IAzureClientFactory<EventGridPublisherClient> clientFactory)
+    {
+        EventGridPublisherClient client = clientFactory.CreateClient("Default");
+
+        client.SendEventAsync(new CloudEvent(...));
+
+        client.SendEventAsync(new EventGridEvent(...));
+
+        // And other overloads...
+    }
+}
+```
+
+Look out for [more information about event publishing](https://eventgrid.arcus-azure.net/Features/publishing-events) and the [Arcus secret store](https://security.arcus-azure.net/features/secret-store).
+
+## Moving towards W3C correlation
+When implementing service-to-service correlation across our Arcus components, we used a hierarchical approach. This approach is made deprecated which means that we will have to move towards a new way of correlation. W3C is the way forward. The interesting thing about Microsoft's event publishing and event subscribing system is that it does not come with an automatic service-to-service correlation like other components. This means that Arcus can be a real added value here.
+
+In v3.3 we made sure that the registered event publisher uses the new W3C correlation and that with a single `traceparent` [custom delivery property](https://learn.microsoft.com/en-us/azure/event-grid/delivery-properties), you could make sure that the event subscriber is correlated to the event publisher. When an event is published and sent toward an Azure Service Bus instance, for example, then Azure Application Insights will make this parent-child relationship visible in the transaction overview.
+
+🚩 Note that the event publisher needs an Arcus correlation implementation. Use [Arcus HTTP correlation](https://webapi.arcus-azure.net/features/correlation), [Arcus Messaging built-in correlation](https://messaging.arcus-azure.net/Features/message-handling/service-bus) or [implement your own](https://observability.arcus-azure.net/Features/correlation).
+
+Look out for [more information about event publishing](https://eventgrid.arcus-azure.net/Features/publishing-events) where we discuss how you can add such a custom delivery property and how this works with other event subscribers.
+
+## Managed identity authentication support
+But that is not all. v3.3 does not only contain a whole new way of event publishing and event correlation, but we also made sure that you can now more easily register such an event publisher with the recommended managed identity authentication system.
+
+Using managed identity during the event publisher registration only requires a topic endpoint, making it not only the most recommended but the easiest way to register event publishers:
```

Comment:
```suggestion
Using managed identity during the event publisher registration only requires a topic endpoint, making it not only the safest but the easiest way to register event publishers:
```
(You already used 'most recommended' in a previous sentence, so I think it makes sense to avoid duplication here and in the meantime also mention why it is the most recommended way).

Reply:
Aha, you're right. Thx for fixing this.